### ISSUE TITLE
Uploader: select all

### ISSUE THREAD

**Issue by tobiasKaminsky** [owncloud/android#1469]
While adding files from a folder on the device, every single file has to be selected one by one.
There should be an option to select all files at once.
@jancborchardt @AndyScherzinger do you have an idea where we can do this?

**Comment by AndyScherzinger** [comment]
Looking at today's upload dialog I would suggest adding it as a menu item but overflown (three dotted).
![screenshot_20160208-104453](https://cloud.githubusercontent.com/assets/1315170/12882318/1d0e46dc-ce51-11e5-8504-77867a8d5265.png)

**Comment by jancborchardt** [comment]
How do other apps do this? As far as I remember some others have a function for »Select all« in the multiselect bar. So as soon as you select the first file, you can select all from there.

**Comment by AndyScherzinger** [comment]
@jancborchardt exactly, but I only found applications that have this in the overflow menu not as text/icon in the bar itself.

**Comment by tobiasKaminsky** [comment]
I think that the "three dot" menu is ok as this function is not that often used?

**Comment by AndyScherzinger** [comment]
Yes, I think so too :+1:

**Comment by jancborchardt** [comment]
Cool, go for it then. :)

**Comment by masensio** [comment]
I think this issue is implemented by  https://github.com/owncloud/android/pull/1342

**Comment by AndyScherzinger** [comment]
Yes, but not in the three dot menu I think, plus the contrib. agreement needs to be signed there.

**Comment by tobiasKaminsky** [comment]
I have rebased it, and I think it might be better to show it direct?
Then we would have (in beta) sorting and select/deselect seen at once.
